Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: build sdk-openssl brupop-image
```

Comment:
Since this is being written in Rust, it would be a good idea to add a license check using `cargo-deny`. That could be added as a separate PR later down the road though.

Reply:
Great tip! I wasn't aware of that tool.

Reply:
I've created issue #87  for this!